modules/fscloud/main.tf

```diff
@@ -1,20 +1,25 @@
 module "event_streams" {
-  source                     = "../../"
-  resource_group_id          = var.resource_group_id
-  es_name                    = var.es_name
-  plan                       = "enterprise-3nodes-2tb"
-  region                     = var.region
-  kms_key_crn                = var.kms_key_crn
-  existing_kms_instance_guid = var.existing_kms_instance_guid
-  schemas                    = var.schemas
-  schema_global_rule         = var.schema_global_rule
-  tags                       = var.tags
-  access_tags                = var.access_tags
-  topics                     = var.topics
-  service_endpoints          = "private"
-  cbr_rules                  = var.cbr_rules
-  service_credential_names   = var.service_credential_names
-  metrics                    = var.metrics
-  quotas                     = var.quotas
-  kms_encryption_enabled     = true
+  source                               = "../../"
+  resource_group_id                    = var.resource_group_id
+  es_name                              = var.es_name
+  plan                                 = "enterprise-3nodes-2tb"
+  region                               = var.region
+  kms_key_crn                          = var.kms_key_crn
+  existing_kms_instance_guid           = var.existing_kms_instance_guid
+  skip_kms_iam_authorization_policy    = var.skip_kms_iam_authorization_policy
+  skip_es_s2s_iam_authorization_policy = var.skip_es_s2s_iam_authorization_policy
```

Comment:
@ocofaigh, should we set this `skip_es_s2s_iam_authorization_policy` as `true` in fscloud example and have permanet policy for es to es so to avoid conflict?